src/Controller/CarteController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Category;
+use App\Repository\CategoryRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class CarteController extends AbstractController
+{
+    /**
+     * @Route("/carte", name="carte")
+     */
+    public function index(): Response
```

Comment:
the name of this function is not adequate, instead use a name such as "menuMap", "carteMenu" ...Etc